packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -340,7 +307,7 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
       // should wait for finish signal if it's a pageload transaction
       disableAutoFinish: isPageloadTransaction,
       beforeSpanEnd: span => {
-        _collectWebVitals();
+        _collectWebVitals?.();
```

Comment:
l: Can we add a comment here that this will always be defined once `beforeSendSpan` triggers? Something along the lines of

```suggestion
       // This function is set in `setup` and will always be defined. TS Just doesn't understand this.
        _collectWebVitals?.();4
```

(alternatively, we could add a non-null assertion and eslint-ignore it with a similar reason.